package.json

```diff
@@ -53,7 +53,7 @@
   },
   "peerDependencies": {
     "react": "^15.3.1",
-    "react-redux": "^4.4.5",
+    "react-redux": "^5.0.4",
```

Comment:
Since this is technically compatible with the entire `4x` branch also, can we please allow both.

Example:
```
"react-redux": "^4.0.0 || ^5.0.0"
```